QRGEN.py

```diff
@@ -10,7 +10,7 @@
 # Modified wrapper template to use gzip decompression instead of deflate
 WRAPPER_TEMPLATE = (
 """<script type="module">\
-document.open();document.write(await new Response(new Response(Uint8Array.from(atob("{b64}"), c => c.charCodeAt(0))).body.pipeThrough(new DecompressionStream("gzip"))).text());document.close();\
+document.open();document.write(await new Response(new Response(Uint8Array.fromHex({bigint}n.toString(16))).body.pipeThrough(new DecompressionStream("gzip"))).text());document.close();\
 </script>"""
```

Comment:
Just to shave off a few more bytes:

```suggestion
"""<script>\
(async(d,R)=>{{\
d.open();\
d.write(await new R(new R(Uint8Array.fromHex({bigint}n.toString(16))).body.pipeThrough(new DecompressionStream("gzip"))).text());\
d.close();\
}})(document,Response)\
</script>"""
```

Reply:
Or... we probably don’t need `document.open()` / `.close()` altogether?

Reply:
This is weird, I can't seem to be able to play it after I put the URI data in the addresses bar

Reply:
Ran about 5 different PolyThink queries, I think the main issue is that the decoded string is longer and hence the browser cuts it off or it's bigint tailing, would love if someone gets this to work though because it's seriously awesome.

Reply:
Hmm, looks like Uint8Array.fromHex is not available in Chrome yet: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array/fromHex

Is there another way to get a bigint into Uint8Array?

Reply:
```js
(async(R,n,a)=>{
  while(n){
    a.push(Number(n&0xffn));
    n>>=8n;
  }
  document.write(await new R(new R(new Uint8Array(a)).body.pipeThrough(new DecompressionStream("gzip"))).text())
)(Response, /* bigint */, [])
```

Reply:
Oh wow! Please make the changes necessary for this and I'll merge it! This is really awesome haha

Reply:
Ah, sorry, I didn't think of testing it in Chrome.

I'll copy notpushkin's version!